applications/dashboard/controllers/api/AddonsApiController.php

```diff
@@ -178,17 +178,17 @@ public function index(array $query) {
     /**
      * Get a single addon.
      *
-     * @param string $addonID The addon ID to get.
+     * @param string $id The addon ID to get.
      * @param array $query The query string.
      * @return array Returns an addon info row.
      * @throws \Garden\Web\Exception\NotFoundException Throws an exception when the addon is not found.
      */
-    public function get($addonID, array $query) {
+    public function get($id, array $query) {
         $this->permission('Garden.Settings.Manage');
 
-        $id = $this->schema([
-            'addonID:s' => 'The ID of the addon.'
-        ], 'in')->setDescription('Get an addon.');
+//        $id = $this->schema([
```

Comment:
Wut? Why did you comment the schema?

Reply:
I forgot to uncomment it. I thought it wasn't used anywhere, and it was  overwriting the $id param ($addonID) right a way.

Reply:
Yeah. You need to rename addonID to id.